samtranslator/translator/translator.py

```diff
@@ -124,6 +124,9 @@ def translate(
 
         self.sam_parser.parse(sam_template=sam_template, parameter_values=parameter_values, sam_plugins=sam_plugins)
 
+        updated_resources = add_embedded_connectors(sam_template=sam_template)
```

Comment:
> del resource["Connectors"]

> del connector["Properties"]["SourceReference"]

I am not a fan of mutating the input parameters without making it super clear. 

couple of possible ways to make it clearer:
1. rename the function and no need to return anything
2. return the connector resources only and have a separate function to strip away "Connectors:"

Reply:
removed the return from the function. Any suggestions on what the function name should be changed to?